src/content/docs/journey-app-slow/create-benchmarks.mdx

```diff
@@ -0,0 +1,171 @@
+
+        To select an appropriate value for the duration condition, one typical practice is to select the 95 percentile duration of the responses for the last 7 or 15 days. Find this duration threshold using the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/), and use it to determine good events for your SLI:
+
+        ```sql
+        SELECT percentile(duration, 95) FROM Transaction WHERE entityGuid = '{entityGuid}' since 7 days ago limit max
+        ```
+
+        **Valid events fields**
+
+        ```sql
+        FROM: Transaction
+        WHERE: entityGuid = '{entityGuid}' AND transactionType = 'Web'
+        ```
+
+        Where `{entityGuid}` is the service's GUID.
+
+        **Good events fields**
+
+        ```sql
+        FROM: Transaction
+        WHERE: entityGuid = '{entityGuid}' AND transactionType = 'Web' AND duration < {duration}
+        ```
+
+        * Where `{entityGuid}` is the service's GUID.
+        * Where `{duration}` is the response time that you consider provides a good experience for your client service or end-user, in seconds.
+    </Collapser>
+    </CollapserGroup>
+
+    <Callout variant="tip">
+    Your organization should define SLOs and SLIs based on your specific needs, your user's expectations, and resources available. After completing this tutorial, we recommend you [learn more about how to define granular custom service levels](/docs/service-level-management/create-slm/#define-user-exp).
+    </Callout>
+    
+    </Step>
+
+    <Step>
+    
+
+    <img
+    title="SLI ui"
+    alt="An image of the service level maangement ui"
+    src={sonicSliUiOverview}
+    />
+
+    1. Navigate to **[one.newrelic.com](https://one.newrelic.com) > All capabilities > Service levels management**. This UI shows all your service levels and allows you to define, monitor, and edit them. 
+    2. Select **+ Add a service level** in the top right of the UI.
+    </Step>
+
+    <Step>
+    Choose the corresponding entity that you want to create a service level for. This could be an entire workload, a specific service, a synthetic monitor, or even a specific transaction. Once you've selected your entity, click **Continue** on the left side of the UI pane.
+    </Step>
+
+    <Step>
+ 
+
+    Define the SLI you chose in step one in this pane. For example if you chose to define an SLI for browser app success, you would use the following queries:
+
+    **Query for valid events**:
+    ```
+    FROM: PageView
+    WHERE: entityGuid = '{entityGuid}'
+    ```
+
+    **Query for bad events**:
+    ```
+    FROM: JavaScriptError
+    WHERE: entityGuid = '{entityGuid}' AND firstErrorInSession IS true
+    ```
+
+    Select **Continue** in the left pane once you've confirmed your queries are correct.
```

Comment:
same comment from before re: alluding to where they're going to go by clicking continue